app/components/ui/Navigation/Navigation.jsx

```diff
@@ -46,27 +57,23 @@ class Navigation extends React.Component {
     // sites, we want to just use an internal react-router Link to the root URL,
     // since 1) this allows us to use react-router routing and 2) this avoids
     // having staging and development environments link to the production site.
-    let logoLink;
-    if (isSFFamiliesSite()) {
-      logoLink = (
-        <a className={styles.navLogoSFFamilies} href={getSiteUrl()}>
-          <img src={images.logoSmall} alt={getSiteTitle()} />
-        </a>
-      );
-    } else {
-      logoLink = (
-        <Link className={styles.navLogo} to="/">
-          <img src={images.logoSmall} alt={getSiteTitle()} />
-        </Link>
-      );
-    }
-
     return (
-      <nav className={isSFFamiliesSite() ? styles.siteNavSFFamilies : styles.siteNav}>
+      <nav className={siteNavStyle}>
         <div className={styles.primaryRow}>
-          <div className={styles.navLeft}>
-            {logoLink}
-            {showSearch
+          {
+            /^https?:\/\//.test(logoLinkDestination)
```

Comment:
I was being creative here by using regex, hopefully this makes sense but it is a string matching the first portion that must be http:// or https:// to return true.

Reply:
I think this is a reasonable thing to do, but it might be more readable if you extracted it out into its own function and gave it a name that was more descriptive, like `isExternalURL`.